### PR TITLE
Helm Chart for pentaho-carte

### PR DESCRIPTION
**Pull request by ejcsid**
**Description**

Helm Chart for pentaho-carte

### CODE REVIEW

File: charts/pentaho-carte/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# ad2image
```
Comment (eidottermihi) [inline charts/pentaho-carte/README.md]: c/p leftover

File: charts/pentaho-carte/templates/tests/test-connection.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```
Comment (eidottermihi) [inline charts/pentaho-carte/templates/tests/test-connection.yaml:1]: it seems like pentaho-carte is not really built for cloud-native... 😢 Without knowing pentaho-carte in detail I also have no better ideas for a test.

if you don't get this test working, you can delete the test file (and folder) and skip the chart testing as a last resort.

**Review by eidottermihi — CHANGES_REQUESTED** [review]

File: charts/pentaho-carte/README.md (hunk 1 above)
Comment (ejcsid) [reply charts/pentaho-carte/README.md]: Thanks - changed it to pentaho-carte.

File: charts/pentaho-carte/templates/tests/test-connection.yaml (hunk 2 above)
Comment (ejcsid) [reply charts/pentaho-carte/templates/tests/test-connection.yaml:1]: decided to keep the test file executing 'curl --help' as either connection fails (wget --header ... ) or command not allowed (curl --user ...), see also comments in test-connection.yaml lines 17ff.

**Pull request merged**